src/CodeGeneration/ApiGenerator/RestSpecification/XPack/MachineLearning/xpack.ml.delete_forecast.patch.json

```diff
@@ -0,0 +1,16 @@
+{
+  "xpack.ml.delete_forecast": {
+    "url": {
+      "path": "/_xpack/ml/anomaly_detectors/{job_id}/_forecast/{forecast_id}",
+      "paths": [
+        "/_xpack/ml/anomaly_detectors/{job_id}/_forecast/{forecast_id}"
+      ],
+      "parts": {
+        "forecast_id": {
+          "required": true,
+          "description": "The ID of the forecast to delete, can be comma delimited list or `_all`"
```

Comment:
Since this can be a comma delimited list, this should maybe map to `Ids` or another, new type?

Reply:
Good point, have implemented a custom type, see: https://github.com/elastic/elasticsearch-net/pull/3591/commits/06474b14a41ed4923adfbf36d496863adb522469